ssdeep.go

```diff
@@ -117,6 +117,14 @@ type Reader interface {
 	io.Reader
 }
 
+var (
```

Comment:
they should be at the top of the file and constants instead.

Reply:
Moved them, but they have to be `var` since non-primitive types can't be `const`.